src/openassetio-core/include/openassetio/hostApi/HostInterface.hpp

```diff
@@ -72,6 +74,8 @@ class OPENASSETIO_CORE_EXPORT HostInterface {
    * specific host in user-facing presentations.
    *
```

Comment:
[consider][minor] This might read better like above, with a "for example:" before the list of examples. Unlikely, but possible someone might think this list is the set of supported hosts or something.

Reply:
Good spot - thanks. Definitely. Added [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/842/commits/d5843ecb70ca8a929e2431799fd88a0035b907fa)